service/awsconfig/v7/drainer_resource_set.go

```diff
@@ -25,27 +26,26 @@ type DrainerResourceSetConfig struct {
 }
 
 func NewDrainerResourceSet(config DrainerResourceSetConfig) (*framework.ResourceSet, error) {
-	if config.GuestAWSClients.CloudFormation == nil {
```

Comment:
I removed the validation here because the configuration is not used, but forwarded, where it is finally validated. No reason to have this in each propagation layer.